tests/unit/testsuite.js

```diff
@@ -305,11 +305,11 @@ window.domEqual = function( selector, modifier, message ) {
 			result = {};
 		$.each( properties, function( index, attr ) {
 			var value = elem.prop( attr );
-			result[ attr ] = value !== undefined ? value : "";
+			result[ attr ] = value != null ? value : "";
```

Comment:
We do the normalization because of space delimited attributes, the most common one being `class`. The attribute will start out `undefined`, but after `elem.addClass( "foo" ).removeClass( "foo" )`, the attribute will be an empty string. We consider this equivalent.

For the property check, the aim was just consistency in the logic. I don't think we've ever consider `null` and `undefined` to be non-equivalent in our check.